coderd/rbac/roles.go

```diff
@@ -677,11 +689,13 @@ type Role struct {
 	// that means the UI should never display it.
 	DisplayName string       `json:"display_name"`
 	Site        []Permission `json:"site"`
-	// Org is a map of orgid to permissions. We represent orgid as a string.
-	// We scope the organizations in the role so we can easily combine all the
-	// roles.
+	// Org is a map of organization IDs to permissions. Grouping by organization
+	// makes roles easy to combine.
 	Org  map[string][]Permission `json:"org"`
 	User []Permission            `json:"user"`
+	// OrgMember is a map of organization IDs to permissions. Grouping by
+	// organization makes roles easy to combine.
+	OrgMember map[string][]Permission `json:"org_member"`
```

Comment:
I wonder if it would be better to have like: (naming tbd)

```golang
	Org map[string]struct {
		Wide []Permission `json:"wide"`
		Member []Permission `json:"member"`
	}
```